File: frontend/Business/tg_business_bot/Models/NotificationGetter.py
from typing import Optional
from tg_business_bot.Models.TgAccount import TgAccount


class NotificationGetter(TgAccount):
    restaurant_id: int

    def __init__(self,
                 id: int,
                 username: str,
                 restaurant_id: int,
                 **kwargs):
        super().__init__(id=id,
                         username=username,
                         restaurant_id=restaurant_id,
                         **kwargs)

File: Domain/Models/NotificationGetter.cs
using System.Text.Json;
using System.Text.Json.Serialization;
using Domain.DataBaseAccess;

namespace Domain.Models;

public class NotificationGetter : TgAccount, ITableDataBase
{
    [JsonPropertyName("restaurant_id")] public long RestaurantId { get; set; }
    
    [JsonPropertyName("is_blocked")] public bool IsBlocked { get; set; }
    
    public NotificationGetter(string username, long? id, long restaurantId, bool isBlocked) : base(id, username)
    {
        RestaurantId = restaurantId;
        IsBlocked = isBlocked;
    }
}